### ISSUE TITLE
Add ability to define number of working days per year

### ISSUE THREAD

**Issue by emedina-rldatix** [franciscobmacedo/remotefreelancept#54]
The simulator is assuming no time off.

Ideally we should be able to take some time off.

**Comment by franciscobmacedo** [comment]
True. This should only be relevant when user chooses monthly or daily income I think

**Comment by igorfp25** [comment]
In this case, the number of business days per month would be equally reduced/increased for each month? I mean, divide the number of business day per year (set by the user) by the number of months `nrMonthsDisplay` also set by the user?
Or would there be another way to handle this?
I understand this would be relevant to calculate the monthly and daily-equilavent income.

For example, if the user set 240 days/year, would this mean 20 business days/mo (when 12 months are set)?
